airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/AbstractSshMssqlSourceAcceptanceTest.java

```diff
@@ -88,14 +91,16 @@ private static Database getDatabaseFromConfig(final JsonNode config) {
             String.format(DatabaseDriver.MSSQLSERVER.getUrlFormatString(),
                 config.get(JdbcUtils.HOST_KEY).asText(),
                 config.get(JdbcUtils.PORT_KEY).asInt(),
-                config.get(JdbcUtils.DATABASE_KEY).asText()),
+                config.get(JdbcUtils.DATABASE_KEY).asText()) + ";encrypt=false",
             SQLDialect.DEFAULT));
   }
 
   @Override
   protected void setupEnvironment(final TestDestinationEnv environment) throws Exception {
     testdb = MsSQLTestDatabase.in(BaseImage.MSSQL_2017, ContainerModifier.NETWORK);
+    LOGGER.info("starting bastion");
     bastion.initAndStartBastion(testdb.getContainer().getNetwork());
+    LOGGER.info("bastion started");
```

Comment:
Any chance of pushing these log messages into the bastion container thing? you're probably doing this in another PR, I'm guessing.